site/_apps/hn-pwa.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 9
+title: 'HN-PWA'
+github-title: 'mozeal/hn-pwa'
+libraries: 
+    - name: 'Polymer'
+scaffolding: 'Polymer CLI & Polymer Starter Kit'
+module-bundling: 'Polymer-build with HTML Imports'
+service-worker: 'Application Shell + data caching with sw-precache'
+perfomance-patterns: 'PRPL, code-splitting for granular loading'
+server-side-rendering: 'None'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Google AppEngine'
+other-details: ''
+authors: 
+    - name: 'Mozeal'
```

Comment:
So in order for the contributor to show up underneath their app, you'll need to make sure the name here matches that in the contributor submission (for example if you wanted to use the name `Mozeal`, you'll have to put that exact string here _and_ in your submission under `_contributors`